dynamallow/types/_schematics.py

```diff
@@ -0,0 +1,27 @@
+from schematics.models import Model as _SchematicsModel
+from schematics.exceptions import ValidationError as _SchematicsValidationError, ModelConversionError as _ModelConversionError
+from schematics import types
+
+from .base import BaseModel as _BaseModel
+from ..exc import ValidationError as _ValidationError
+
+
+class Model(_SchematicsModel, _BaseModel):
+    """``Model`` is the base class for schematics_ based schemas """
+    @staticmethod
+    def field_to_dynamo_type(field):
+        """Given a schematics field object return the appropriate Dynamo type character"""
+        if isinstance(field, types.NumberType):
```

Comment:
Might be good to add `# XXX: Schematics does not currently have a "raw" type that would map to Dynamo's 'B' (binary) type.`